### PR TITLE
SUBMARINE-397. [WEB]Implement the job page in workbench with Angular

### PR DESCRIPTION
**Pull request by jasoonn**
### What is this PR for?
Implement the job page in workbench with Angular.

### What type of PR is it?
[Feature]

### Todos
* [ ] - Task

### What is the Jira issue?
https://issues.apache.org/jira/browse/SUBMARINE-397

### How should this be tested?
https://travis-ci.org/apache/submarine/builds/659683029?utm_source=github_status&utm_medium=notification

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by jasoonn** [comment]
@kevin85421 @liuxunorg  Can you help me review the pull request? Thanks!

**Comment by liuxunorg** [comment]
@jasoonn Thank you contribution for this feature.
Because `submarine-test/e2e` has been changed to `submarine-test/test-e2e`, you need to execute the following rebase command:
```
git checkout master
git pull
git checkout SUBMARINE-397
git rebase upstream/master
```

File: submarine-test/test-e2e/src/test/java/org/apache/submarine/integration/jobIT.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.testng.Assert;
+
+public class jobIT extends AbstractSubmarineIT {
+
+  public final static Logger LOG = LoggerFactory.getLogger(sidebarIT.class);
```
Comment (liuxunorg) [inline submarine-test/test-e2e/src/test/java/org/apache/submarine/integration/jobIT.java]: Need change to `public final static Logger LOG = LoggerFactory.getLogger(jobIT.class);`

**Comment by jasoonn** [comment]
@liuxunorg Thank for your advice. I have updated my pr.

**Comment by liuxunorg** [comment]
Will merge if no more comments

**Pull request closed without merging**